### ISSUE TITLE
META6.json URI version "v0.1.4" instead of "0.1.4"?

### ISSUE THREAD

**Issue by CurtTilmes** [perl6-community-modules/uri#40]
Why does `META6.json` have version with a 'v' prefix?  Everything I read in docs ([docs](https://docs.perl6.org/language/modules#index-entry-META6.json-META6.json), [Design, S11](https://design.perl6.org/S11.html#Versioning) seems to recommend it as numbers without the 'v' prefix.

If I run `zef info URI`, it reports:
```
- Info for: URI
- Identity: URI:ver<.0.1.4>
...
```
but when I try to install it with that identity, it fails:
```
zef install --force "URI:ver<.0.1.4>"
===> Searching for: URI:ver<.0.1.4>
No candidates found matching identity: URI:ver<.0.1.4>
```

**Comment by Altai-man** [comment]
I have rights to update the current version with e.g. `0.1.5`, will it be enough? Just on github though, not possible e.g. CPAN release.

**Comment by CurtTilmes** [comment]
The more I look at this, I think it is a real `zef` bug as well (triggered by the malformed version in `URI`).

If my module depends on `URI`, and I run `zef depends .` it says `URI:ver<.0.1.4>`, but that isn't a valid identity I can hand back into e.g. `zef info`.

**Comment by CurtTilmes** [comment]
`zef` says `URI` is hosted on github, so I think if you re-released it with a proper version 0.1.5 everything would work, even without a CPAN release.  (Greatly appreciated since it is breaking my integration checks..)  I filed a `zef` issue to see if that could be patched to prevent issues like this in the future.

**Comment by jonathanstowe** [comment]
```Test::META``` actually tests for this very thing.

**Comment by CurtTilmes** [comment]
Would be cool if every module used `Test::META` on their own metadata.  Could (should?) `zef` check `META6.json` with `Test::META` and fail the install if the metadata is bad?

**Comment by Altai-man** [comment]
I have bumped the version with https://github.com/perl6-community-modules/uri/commit/b645ffdcf02ee685d2c438673b8c9c37a4375d9c
And made a release - https://github.com/perl6-community-modules/uri/releases/tag/0.1.5
Please, close the issue if this helps, or suggest what else can be done to improve the situation. :)

**Comment by Altai-man** [comment]
Also, it is probably just me, but I wonder if `0.1.5` is too shy. I mean, `0.1` is not really a big version for something that is apparently used in the ecosystem relatively often. Didn't review the library though.

**Comment by CurtTilmes** [comment]
That fixed my problem -- it picked up 0.1.5.

Thanks!
